### ISSUE TITLE
The slider (range) color in dark mode is the incorrect color

### ISSUE THREAD

**Issue by coliff** [trimble-oss/website-modus-bootstrap.trimble.com#225]
https://modus-bootstrap.trimble.com/components/sliders/#basic-range

Slider color is `#0063a3` It should be Highlight Blue `#019aeb`.

Thanks for reporting @soniakaukonen !

**Comment by coliff** [comment]
This issue wsa incorrectly marked as 'Done', but it's not fixed yet. We need to add this to dark-mode CSS

```css
.custom-range::-webkit-slider-runnable-track {
  background-color: #019aeb !important;
}

.custom-range::-moz-range-track {
  background-color: #019aeb !important;
}
```

**Comment by coliff** [comment]
PR open: https://bitbucket.trimble.tools/projects/TMDS/repos/modus-bootstrap/pull-requests/285/overview

**Comment by coliff** [comment]
Fixed in v1.5.5
